packages/seacas/scripts/exodus3.in.py

```diff
@@ -5781,6 +5818,17 @@ def __ex_get_var(self, timeStep, varType, varId, blkId, numValues):
             var_vals)
         return var_vals
 
+    def __ex_get_var_time(self, varType, varId, entityID, start_step, end_step):
+        s_step = ctypes.c_int(start_step)
+        e_step = ctypes.c_int(end_step)
+        var_type = ctypes.c_int(get_entity_type(varType))
+        var_id = ctypes.c_int(varId)
+        entity_id = ctypes.c_longlong(entityID)
+        time_range = range(start_step, end_step + 1)
```

Comment:
The use of `time_range` and then using the length of it seems more convulted than just `num_steps = end_step - start_step + 1` and then `var_vals = (ctypes.c_double * num_steps)()`

Reply:
I agree, that does seem like the simpler option.
Change has been made in latest push.